EventMixing/src/MixBackgroundFrames_module.cc

```diff
@@ -92,6 +98,16 @@ namespace mu2e {
           Comment("Write out IDs of events on the secondary input stream."),
           false
           };
+
+      fhicl::Sequence<std::string> simStageEfficiencyTags{ Name("simStageEfficiencyTags"),
+          Comment("Sequence of strings for all the previous simulation stage efficiencies that need to be included"),
+          std::vector<std::string>()
+          };
+
+      fhicl::Sequence<double> meanEventsPerPOTFactors{ Name("meanEventsPerPOTFactors"),
```

Comment:
Are meanEventsPerPOTFactors some fudge factors that should approximately never be used?   It should not be a required piece of configuration, make it an OptionalSequence.